%YAML:1.0
---
cameraMatrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1.3657274540559017e+03, 0., 9.6448887315135164e+02, 0.,
       1.3696772835310642e+03, 5.4261110086904466e+02, 0., 0., 1. ]
distCoeffs: [ 3.9793163276328640e-02, 1.0891609115693963e+00,
    -2.1160009562345219e-03, -3.7623769372493519e-03,
    -5.3020145910777448e+00 ]
